coding-concepts.js

```diff
@@ -6,54 +6,65 @@
 
 // --------------------1) What will this log?
+10
 
-const cohort = "LEARN 2022"
+// const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
-// a) Your answer:
+// a) Your answer: 10 
 // b) Verify and explain:
+// My answer was correct as I predicted the output would come out as 10. Looking at this, I can see that the variable is being assigned a data type of a string with the information of "LEARN 2022". I can see there is a .length within our code. .length() is a informational command that returns the number of elements in the array. By utilizing function console.log(), we are asking the computer to recall the declared variable cohort and use the built-in method .length to output the amount of elements used within the string. "LEARN 2022" has an index amount of 9. Because .length counts the elements from 0, it adds 1 to the last index. This is why I knew the output would be 10. 
 
 
 // --------------------2) What will this log?
 
-const greeting = "Hello World!"
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
+// a) Your answer: o
 // b) Verify and explain:
-
+// My answer of o was correct when I ran my code. Looking at this code, I can see that variable was assigned to a string of "Hello World!". The function of console.log() is calling for the computer to review the variable of greeting. Seeing that within the parentheses, followed after greeting, are square brackets []. These square brackets indicate an array which are zero indexed. This indicated to me that we are asking for the computer to find the index of 4 within the data type of string. As arrays are zero indexed, you must start your count with the first element as 0 and work your way up. When you do this, you will find that the letter "o" index is 4. 
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
+// a) Your answer: Ruby
 // b) Verify and explain:
-
+// Before running this code, I predicted that the answer/output would be Ruby which was correct. Looking at this code, I can see that the variable languages was assigned an array of four different strings as elements. There is another variable named index that is assigned a number of 1. Looking at the function of console.log(), it is calling for the computer to review the array in the variable languages and look for the element that has the location of 1.  I know we are looking for the location of 1 because within the function console.log(), we are also calling for the second variable index. As arrays are zero indexed, the computer reviewed the array and started counting the elements starting from zero. Thus the output was Ruby since it has the location of 1. 
```

Comment:
I think I understood what you meant. just for clarity...
`we are also calling for the second variable index.` let's rephrase this to calling on the index of the second value in the array because the variable "index" is assigned the number 1